docker/celery-pyfr/Dockerfile

```diff
@@ -6,26 +6,26 @@ LABEL maintainer="patrick.oleary@kitware.com" \
       pyfr.version="1.8.0"
 
 RUN apt-get update && \
-    apt-get install -y build-essential \
-                       gcc-6 \
-                       g++-6 \
-                       g++-6-multilib \
-                       gfortran-6 \
-                       libopenblas-dev \
-                       liblapack-dev \
-                       openssh-client \
-                       openssh-server \
-                       openmpi-bin \
-                       libopenmpi-dev \
-                       libhdf5-dev \
-                       libmetis-dev \
-                       strace \
-                       unzip \
-                       wget \
-                       --no-install-recommends && \
+    apt-get install -y --no-install-recommends \
```

Comment:
was dropping build-essential intentional?

Reply:
I just moved it to the base image (`bionic-python`).